OpenSim/Simulation/Test/testInitState.cpp

```diff
@@ -50,8 +50,10 @@ int main()
     try {
         LoadOpenSimLibrary("osimActuators");
         testStates("arm26.osim");
-        testMemoryUsage("arm26.osim");
-        testMemoryUsage("PushUpToesOnGroundWithMuscles.osim");
+        if (isGetRSSValid()) {
```

Comment:
I tend to agree @tkuchida. The flip-side is: should CI testing *fail* because `getRSS` is flaking out?

I do prefer to have the explicit exception as you suggest and then the reviewers could make the call to merge a change or not based on the test passing/failing on other platforms. 

If we do break validity of getRSS out into a separate test it may be possible that `isGetRSSValid()` is then true but had it been evaluated in another process (the actual memory test) it would have been false. Then, another option is to bake the `isGetRSSValid()` check into every memory evaluation and throw a `getRSS() is not currently valid` exception.  As a reviewer you would not have to determine if the test failed because of a legitimate leak or some quirk with getRSS() - our instrumentation would tell you it was getRSS(). It could potentially result in more failures, but it would be more clear which ones can be ignored.

I mainly want to avoid having to rebuild PRs just to get these memory tests to pass on all platforms. If it continues to be a problem, we should consider just having it on Windows and/or use other tools on Linux/Mac.